platform/broadcom/raw-image.mk

```diff
@@ -0,0 +1,17 @@
+# sonic broadcom raw image installer
+
+SONIC_RAW_IMAGE = sonic-broadcom.raw
+$(SONIC_RAW_IMAGE)_MACHINE = broadcom
+$(SONIC_RAW_IMAGE)_IMAGE_TYPE = raw
+$(SONIC_RAW_IMAGE)_DEPENDS += $(BRCM_OPENNSL_KERNEL)
+$(SONIC_RAW_IMAGE)_INSTALLS += $(DELL_S6000_PLATFORM_MODULE) \
```

Comment:
it is better to make raw_image depends on one_image, so that you do not need to duplicate these dependencies.